gtsfm/runner/run_scene_optimizer_colmaploader.py

```diff
@@ -0,0 +1,62 @@
+import argparse
```

Comment:
maybe have a config for the loader too, instead of creating copies of the file?

Reply:
or an if-else structure where we take the name of the loader from the command line args?

Reply:
i think each loader will have different set of required arguments (at least the way I've defined it). So I think it makes sense for them to be separate. Can you take a look at the 3 runner files in this branch and check it out?